gdal/doc/source/drivers/vector/flatgeobuf.rst

```diff
@@ -39,6 +39,17 @@ Layer Creation Options
 -  **SPATIAL_INDEX=**\ *YES/NO*: Set the YES to create a
    spatial index. Defaults to YES.
 
+Examples
+--------
```

Comment:
Nitpicking: I think it's safer to assume reST expects heading markup anf paragraph are separated by a blank line